students/randi_peterson/session05/mailroom.py

```diff
@@ -0,0 +1,92 @@
+import os.path as path
+
+def thank_you(donor_dict):
+    #Record single donation and thank donor
+    fullname = input('What is the full name of the donor? Type list to see donors. Type exit to quit task \n')
+    if fullname.lower() == 'list':
+        print(list(donor_dict.keys()))
+    elif fullname.lower() == "exit":
+        return None
+    else:
+        #Find donor in list or create a new one
+        if fullname not in donor_dict:
+            donor_dict[fullname] = []
+        newdonation = float(input('What is the value of the donation (numbers only)?\n'))
+        donor_dict[fullname].append(newdonation)
+
+        #Print the thank you email
+        print(f'Dear {fullname}, \n Thank you for your generous donation. We appreciate the support from people like you. \n Thank you,\n Charity Name')
```

Comment:
I notice you're leaving the donation amounts out of the thank you letters and responses.  Is there a reason?